mne_bids/write.py

```diff
@@ -112,8 +112,7 @@ def _channels_tsv(raw, fname, overwrite=False):
 
     # get the manufacturer from the file in the Raw object
     _, ext = _parse_ext(raw.filenames[0])
-    manufacturer = MANUFACTURERS[ext]
-
+    manufacturer = MANUFACTURERS.get(ext, '')
```

Comment:
yessss, this looks good. exactly what I had in mind.

Reply:
to test this, we could actually recycle my minimal example into a unit test. WDYT?